.github/workflows/build_wheels.yml

```
@@ -0,0 +1,61 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    env:
+      GEOS_VERSION: "3.9.1"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-2019]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: ilammy/msvc-dev-cmd@v1
+        with:
+          arch: x64  # TODO: 32-bit version (x86)
+        if: ${{ matrix.os == 'windows-2019' }}
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v1.12.0
+        env:
+          CIBW_BUILD: cp38-*
+          CIBW_ARCHS: auto64  # TODO, 32-bit builds (especially on windows)
+          CIBW_ENVIRONMENT_LINUX:
+            GEOS_VERSION=${{ env.GEOS_VERSION }}
+            GEOS_INSTALL=${{ github.workspace }}/geos-${{ env.GEOS_VERSION }}
+            GEOS_CONFIG=${{ github.workspace }}/geos-${{ env.GEOS_VERSION }}/bin/geos-config
+            LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${{ github.workspace }}/geos-${{ env.GEOS_VERSION }}/lib
+          CIBW_ENVIRONMENT_MACOS:
+            GEOS_VERSION=${{ env.GEOS_VERSION }}
+            GEOS_INSTALL=${{ github.workspace }}/geos-${{ env.GEOS_VERSION }}
+            GEOS_CONFIG=${{ github.workspace }}/geos-${{ env.GEOS_VERSION }}/bin/geos-config
+            DYLD_FALLBACK_LIBRARY_PATH=${{ github.workspace }}/geos-${{ env.GEOS_VERSION }}/lib
+            LDFLAGS=-Wl,-rpath,${{ github.workspace }}/geos-${{ env.GEOS_VERSION }}/lib
+          CIBW_ENVIRONMENT_WINDOWS:
+            GEOS_VERSION=${{ env.GEOS_VERSION }}
+            GEOS_INSTALL='${{ github.workspace }}\geos-${{ env.GEOS_VERSION }}'
+            GEOS_LIBRARY_PATH='${{ github.workspace }}\geos-${{ env.GEOS_VERSION }}\lib'
+            GEOS_INCLUDE_PATH='${{ github.workspace }}\geos-${{ env.GEOS_VERSION }}\include'
+            PATH=$PATH';${{ github.workspace }}\geos-${{ env.GEOS_VERSION }}\lib'
```

`geos_c.dll` is in the `bin` folder. The `lib` folder is not useful for PATH (I think it's for [linker inputs](https://docs.microsoft.com/en-us/cpp/build/reference/dot-lib-files-as-linker-input?view=msvc-160)).
```suggestion
            PATH=$PATH;${{ github.workspace }}\geos-${{ env.GEOS_VERSION }}\bin
```
also, I'm guessing single quotes as not necessary for PATH (but keep them for the other variables).